src/util.h

```diff
@@ -249,4 +249,6 @@ template <typename Callable> void TraceThread(const char* name,  Callable func)
 
 std::string CopyrightHolders(const std::string& strPrefix);
 
+/* returns the current SSL product and version info */
```

Comment:
Looks like this file is completely documented, can you please extend the comments? E.g. like this:

```c++
/**
  * Return the product name with the version number of the SSL library being used.
  * @note Bitcoin Core supports OpenSSL and LibreSSL.
  */
```